### PR TITLE
fix puppetdb version setting

### LINKED ISSUE #44: 3.1.1 not starting up
```
2017-08-30T14:11:32 2bfa4e39 [app] [I] Completed 500 Internal Server Error in 126ms (Views: 3.3ms | ActiveRecord: 4.7ms)
2017-08-30T14:11:32 280827f6 [app] [I] Started GET "/settings" for 2a00:cb0:8002:1055:6af7:28ff:fec1:b973 at 2017-08-30 14:11:32 +0200
2017-08-30T14:11:32 280827f6 [app] [I] Processing by SettingsController#index as HTML
2017-08-30T14:11:32 280827f6 [app] [I] Current user: mmoll (administrator)
2017-08-30T14:11:32 280827f6 [app] [I]   Rendered settings/category/_email.html.erb (0.7ms)
2017-08-30T14:11:32 280827f6 [app] [I]   Rendered settings/index.html.erb within layouts/application (161.4ms)
2017-08-30T14:11:32 280827f6 [app] [W] Action failed
| NameError: uninitialized constant PuppetDB
| /usr/share/foreman/vendor/ruby/2.3.0/gems/puppetdb_foreman-3.1.1/app/models/setting/puppetdb.rb:34:in `block in default_settings'
| /usr/share/foreman/app/models/setting.rb:269:in `block (2 levels) in set'
| /usr/share/foreman/app/helpers/settings_helper.rb:7:in `value'
| /usr/share/foreman/app/views/settings/index.html.erb:30:in `block (2 levels) in _693c7dd7edb049ca322cd91a6fdc4573'
| /usr/share/foreman/app/views/settings/index.html.erb:27:in `each'
| /usr/share/foreman/app/views/settings/index.html.erb:27:in `block in _693c7dd7edb049ca322cd91a6fdc4573'
| /usr/share/foreman/app/views/settings/index.html.erb:15:in `each'
| /usr/share/foreman/app/views/settings/index.html.erb:15:in `_693c7dd7edb049ca322cd91a6fdc4573'
```

### PR DESCRIPTION
**Pull request by timogoebel**
Fixes #44.

### CODE REVIEW

File: app/services/puppetdb.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module Puppetdb
   API_VERSIONS = {
-    '4' => 'v4: PuppetDB 2.3, 3.0, 3.1, 3.2, 4.0 (PE 3.8, 2015.2, 2015.3)',
-    '3' => 'v3: PuppetDB 1.5, 1.6 (PE 3.1, 3.2, 3.3)',
-    '2' => 'v2: PuppetDB 1.1, 1.2, 1.3, 1.4'
+    '4' => 'v4: PuppetDB 4.0, 4.1, 4.2, 4.3',
```
Comment (mmoll) [inline app/services/puppetdb.rb]: also 5.x? https://docs.puppet.com/puppetdb/5.0/api/index.html#version-4

**Comment by mmoll** [comment]
fixes the problem for me 👍

File: app/services/puppetdb.rb (hunk 1 above)
Comment (timogoebel) [reply app/services/puppetdb.rb]: I haven't tested 5.x so I don't want to list it officially.

**Comment by timogoebel** [comment]
Merged, thanks @mmoll.

**Pull request merged**